app/views/practices/form/metrics.html.erb

```diff
@@ -31,6 +40,18 @@
               <span>Print</span>
             </a>
           </div>
+          <div class="margin-top-2 margin-bottom-2">
+            <% if params.has_key?(:session_locked) %>
+              <div class="usa-alert usa-alert--warning margin-bottom-2 message-icon-bg-position">
+                <div class="usa-alert__body grid-row flex-align-center">
+                  <div class="usa-alert__heading font-sans-sm x0-bottom"><%= params[:session_locked] %></div>
+                </div>
+              </div>
```

Comment:
Can we turn this alert div into a partial, since we are using something similar in the code above?

Reply:
Done